framework/templates/ecl-pages/ecl-pages.twig

```diff
@@ -41,77 +41,6 @@
 
 {% include '@ec-europa/ecl-footers' with {
   footer_menus: [
-    {
-      type: 'branded',
-      wrapper_class: 'ecl-footer__site-identity',
```

Comment:
Why did you remove the site identity from this example?

Reply:
copy/paste... Didn't saw that